src/main/webapp/assets/js/util/brooklyn-utils.js

```diff
@@ -113,6 +113,7 @@ define([
         if ($input.attr("type") === "checkbox") {
             return $input.is(":checked");
         } else {
+            // TODO: get codemirror.getValue() in case this is a textarea ...
```

Comment:
can you elaborate?  (or just say something like "comment from @azbarcea, not clear what it means but left in case that becomes clear one day")